charts/flyte-binary/values.yaml

```diff
@@ -119,6 +119,28 @@ configuration:
       repository: cr.flyte.org/flyteorg/flytecopilot # FLYTECOPILOT_IMAGE
       # tag CoPilot sidecar image tag
       tag: v0.0.28 # FLYTECOPILOT_TAG
+  flyte-binary:
```

Comment:
Could this block be moved under the `configuration` block instead? `flyte-binary` might not be necessary here.